avogadro/rendering/geometryvisitor.h

```diff
@@ -62,6 +64,22 @@ class GeometryVisitor : public Visitor
    */
   float radius();
 
+  /**
+   * <<API Extension for TDX>>
```

Comment:
Please, please omit "API Extension." I'm more than happy to provide credit / acknowledgement for your work and TDX, but we don't need to clutter the documentation with who wrote particular methods.